File: 231-power-of-two/power-of-two.cpp
class Solution {
public:
    bool isPowerOfTwo(int n) {
        int sum=0;
        if(n<0){return 0;}
        if(n==1){return true;}
        if(n==0){return 0;}

        while(n>0){
            sum=sum+(n%2);
            n=n/2;          
             
        }
        
        if(sum==1){return true;}
        
         else{   return false;} 
    }
};

File: 231-power-of-two/power-of-two.py
class Solution(object):
    def isPowerOfTwo(self, n):
        if(n<0):
            return False
        if(n==0):
            return False
        if(n==1):
            return True
        
        sum=0
        while(n>0):
            sum=sum+(n%2)
            n=n//2
        if(sum==1):
            return True
        else:
            return False
        return False
        